lib/src/Uncertainty/Distribution/NormalFactory.cxx

```diff
@@ -87,4 +89,26 @@ Normal NormalFactory::buildAsNormal() const
   return Normal();
 }
 
+
+DistributionFactoryResult NormalFactory::buildEstimator(const NumericalSample & sample) const
+{
+  Normal distribution(buildAsNormal(sample));
+  NumericalPoint mu(distribution.getMean());
+  NumericalPoint sigma(distribution.getSigma());
+  const UnsignedInteger size = sample.getSize();
+  const UnsignedInteger dimension = distribution.getDimension();
+  NumericalPoint parameter(distribution.getParameter());
+  ComposedDistribution::DistributionCollection coll(parameter.getSize());
+  for (UnsignedInteger j = 0; j < dimension; ++ j)
```

Comment:
No the asymptotic estimate is for dimension 1 only
For upper dimension it is more complex (Wishart product...)